include/nbl/builtin/hlsl/loader/mtl/common.hlsl

```diff
@@ -0,0 +1,43 @@
+
+// Copyright (C) 2018-2022 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_BUILTIN_HLSL_MTL_LOADER_COMMON_INCLUDED_
+#define _NBL_BUILTIN_HLSL_MTL_LOADER_COMMON_INCLUDED_
+
+
+namespace nbl
+{
+namespace hlsl
+{
+
+
+struct MTLMaterialParameters
```

Comment:
make the namespace deeepaaaaah! `nbl::hlsl::loader::mtl` and just call the struct `Parameters`